source/renderer/app/components/sidebar/wallets/SidebarWalletMenuItem.js

```diff
@@ -40,6 +43,11 @@ export default class SidebarWalletMenuItem extends Component<Props> {
           <span className={styles.title}>{title}</span>
           <span className={styles.info}>{info}</span>
           {isRestoreActive ? <ProgressBar progress={restoreProgress} /> : null}
+          {isLegacy && (
+            <div className={styles.legacyBadge}>
```

Comment:
@yakovkaravelov why do we need a `<div className={styles.legacyBadge}>` wrapper here? Can't we move all the needed HTML elements into the `<LegacyBadge />` itself?